.github/workflows/bc.yml

```diff
@@ -3,6 +3,16 @@ name: bc
 on:
   workflow_call:
     inputs:
+      composer-root-version:
+        description: Composer root version.
+        default: dev-master
+        required: false
+        type: string
+      subpackage:
+        description: Subpackage to install.
+        default: ''
+        required: false
+        type: string
```

Comment:
Should it be possible to install multiple subpackages or would it always be a single one?

Reply:
This will be great to find a way to install multiple subpackages but I don't see a way how to loop `uses` construction.
I see only a way to do it by enumeration (e.g. `install subpackage1`, `install subpackage2` or `install subpackages[0]`, `install subpackages[1]`). Any suggestions? Maybe we can realize this in the future.

Reply:
Done

Reply:
Done via enumeration for 5 packages